connectors/sources/notion.py

```diff
@@ -28,7 +28,7 @@
 RETRY_INTERVAL = 2
 DEFAULT_RETRY_SECONDS = 30
 BASE_URL = "https://api.notion.com"
-MAX_CONCURRENT_CLIENT_SUPPORT = 3
+MAX_CONCURRENT_CLIENT_SUPPORT = 30
```

Comment:
This seems like a large change and looks unrelated. What's the rationale?

Reply:
The downloading of the attachments happens via `aiohttp` session using public urls which expire in 1 hour, also those aren't restricted by Notion's rate limit of 3, so at a time more connections can be done for downloading files.

Hence following changes were made for `MAX_CONCURRENT_CLIENT_SUPPORT` and `concurrent_downloads` that is specifically used for downloading of files only.

Should I raise a separate PR for this ?

Reply:
No, I'm fine with you including small fixes in larger PRs. In the future, just try to call them out either with code comments or in the PR description, so it's clear what's being changed and why. 👍